nemo_curator/download/commoncrawl.py

```diff
@@ -280,6 +353,7 @@ def download_common_crawl(
     start_snapshot: str,
     end_snapshot: str,
     output_type: str = "jsonl",
+    algorithm: str = "justext",
```

Comment:
We will keep jusText as the default for this PR. If our research finds it beneficial to switch to Resiliparse, we can update this line to set Resiliparse as the default.